registry/txt_test.go

```diff
@@ -60,6 +60,26 @@ func testTXTRegistryNew(t *testing.T) {
 	_, err = NewTXTRegistry(p, "txt", "txt", "owner", time.Hour)
 	require.Error(t, err)
 
+	_, err = NewTXTRegistry(p, "@invalid#", "", "owner", time.Hour)
+	require.Error(t, err, `invalid TXT prefix provided, expected "[a-z0-9-.]+" got "@invalid#"`)
```

Comment:
This message and the one on like 66 are not accurate, I recommend changing it like: 
```suggestion
	require.Error(t, err)
```

Reply:
Why are they not accurate? By comparing the error string I make sure that the guard that returns the error works. If I just accept _any_ error, it would make it less good, right?

Reply:
Because what is happening when you have an error is not that the TXT prefix is invalid, rather than it is expected to be invalid, but there is no error. The string you are providing is only changing the messaging to report, not changing any behavior at all. The default message instead, is correct.

Reply:
@ruudk any thoughts on the suggestion from @Raffo